test/stdlib/Symbol_test.rb

```diff
@@ -186,7 +186,7 @@ def test_inspect
   end
 
   def test_intern
-    assert_send_type "() -> self",
+    assert_send_type "() -> Symbol",
                      :a, :intern
   end
```

Comment:
I want to add a test to check like this: `def foo: (Symbol | String) -> Symbol`.
But I don't know how to add it, so could you please tell me?

Reply:
I don't think there is a handy way to do that. Having no test sounds reasonable for this case to me.

Reply:
Make sense. Thank you! 👍